doc/release-notes.md

```diff
@@ -119,6 +119,11 @@ Command-line options
 Please check `Help -> Command-line options` in Qt wallet or `dashd --help` for
 more information.
 
+Updated settings
+----------------
+
+- Passing an invalid `-rpcauth` argument now cause dashd to fail to start.
+
```

Comment:
we cannot update release notes like this

Reply:
Moved to `release-notes-5112.md`